src/classes/CW_AssociateAudienceRewardHelperTest.cls

```diff
@@ -0,0 +1,124 @@
+@isTest
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.